frontend/src/scenes/data-warehouse/editor/multitabEditorLogic.tsx

```diff
@@ -339,7 +361,7 @@ export const multitabEditorLogic = kea<multitabEditorLogicType>([
                 return {
                     query: props.monaco?.editor.getModel(model.uri)?.getValue() || '',
                     path: model.uri.path.split('/').pop(),
-                    view: model.view,
```

Comment:
Maybe a mis delete? We'll want the view to still be here otherwise the views won't persist between refreshes